src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1111,6 +1142,50 @@ void FancyZones::OnEditorExitEvent() noexcept
     JSONHelpers::FancyZonesDataInstance().SaveFancyZonesData();
 }
 
+const std::vector<HMONITOR> FancyZones::GetMonitorsSorted()

```

Comment:
we don't need `const` here, but we need `std::shared_lock readLock(m_lock);`

Reply:
OK, yes we need it here. Can we remove it than from `OnSnapHotkey` ?

Reply:
sure